.gitignore

```diff
@@ -20,6 +20,7 @@ queryplan.png
 .lqp*.dot
 lqp*.png
 compile_times.txt
+exported_table_statistics.json
```

Comment:
Attaching this to a random line:
```
[:~/Projekte/Opossum/Git] lawben/general_benchmark(+2/-1)+* ± ./build/hyriseBenchmarkRunner --queries=test2 --tables=test3
[PERF] Hyrise is running as a debug build. at src/lib/utils/performance_warning.cpp:13
	Performance can be affected. This warning is only shown once.

libc++abi.dylib: terminating with uncaught exception of type boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::out_of_range> >: bimap<>: invalid key
Abort trap: 6
ca[:~/Projekte/Opossum/Git] lawben/general_benchmark(+2/-1)+* ± cat test2/bla.sql
SELECT * FROM bla;

[:~/Projekte/Opossum/Git] lawben/general_benchmark(+2/-1)+* ± cat test3/bla.tbl
bla
1
2
3
```

Reply:
the .tbl file is malformed. it should be

```
bla
int
1
2
3
``` 

the `int` is missing

Reply:
Argh. I originally tested it with a csv file and forgot that part.